app/views/works/main.html.erb

```diff
@@ -0,0 +1,65 @@
+<main class="row">
+  <div>
+    <h3>
+      <span class="media-spotlight-header"></span>
+      Media Spotlight:
+      <% if @works.first != nil %>
+      <%= link_to("#{Work.most_popular.title}", work_path(Work.most_popular)) %>
+      by <%= Work.most_popular.creator %>
+      <% else %>
+      <%= link_to("Add some works!", new_work_path) %>
+      <% end %>
+    </h3>
+    <p><%= "#{Work.most_popular.votes.count} votes" %>
+      <% if Work.most_popular.description != nil %>
+      <%= "- #{Work.most_popular.description}"%>
+      <% end %>
+    </p>
+  </div>
+
+  <hr class="root-hr">
+
+  <section class="top-ten-container">
+    <section class="top-ten">
+      <h3 class="top-ten-category">Top Movies</h3>
+      <ul>
+        <% Work.top_ten_movies.each do |w| %>
+        <li>
+          <h4><%= link_to("#{w.title}", work_path(w)) %> <small>by <%= w.creator %></small></h4>
+        </li>
+        <p>
+          <%= "#{w.votes.count} votes" %>
+        </p>
+        <% end %>
+      </ul>
+    </section>
+
+    <section class="top-ten">
+      <h3 class="top-ten-category">Top Books</h3>
+      <ul>
+        <% Work.top_ten_books.each do |w| %>
+        <li>
+          <h4><%= link_to("#{w.title}", work_path(w)) %> <small>by <%= w.creator %></small></h4>
+        </li>
+        <p>
+          <%= "#{w.votes.count} votes" %>
+        </p>
+        <% end %>
+      </ul>
+    </section>
+
+    <section class="top-ten">
+      <h3 class="top-ten-category">Top Albums</h3>
+      <ul>
+        <% Work.top_ten_albums.each do |w| %>
+        <li>
+          <h4><%= link_to("#{w.title}", work_path(w)) %> <small>by <%= w.creator %></small></h4>
```

Comment:
Similar comment regarding string interpolation, you can just put `w.title` directly in the first parameter without listing it explicitly in the string interpolation